public/locales/en/common.json

```diff
@@ -88,5 +89,7 @@
   "sort-descending": "Sort Descending",
   "skip-to-main-content": "Skip to main content",
   "outcome-sub-nodes": "Subsectors",
-  "outcome-sub-nodes-secondary": "Subsectors"
+  "outcome-sub-nodes-secondary": "Subsectors",
+  "outcome-bar-summary-historical": "{{goalType}} in year {{selectedYear}} was {{selectedYearDifference}} different than the goal in year {{nearestGoalYear}} ({{nearestGoalValue}}).",
+  "outcome-bar-summary-forecast": "With the chosen scenario of {{scenarioName}} the forecast of {{goalType}} in {{selectedYear}} is {{selectedYearValue}}. Target for the year {{nearestGoalYear}} is {{nearestGoalValue}}."
```

Comment:
Nitpick: There's some double punctuation going on here, we could strip trailing fullstops before passing them to this translation. No need to do this now though.

<img width="549" alt="Screenshot 2023-10-18 at 9 55 46" src="https://github.com/kausaltech/kausal-paths-ui/assets/15343658/5fb8f6c4-4b06-41a7-b29a-373c6dd1aa08">

Reply:
Yeah noticed that too. I thought as well to leave the trimming to later times.